lib/internal/loader/ModuleRequest.js

```diff
@@ -19,6 +19,8 @@ const search = require('internal/loader/search');
 const asyncReadFile = require('util').promisify(require('fs').readFile);
 const debug = require('util').debuglog('esm');
 
+const esModuleSymbol = exports.esModuleSymbol = Symbol('esModule');
```

Comment:
I would prefer `Symbol.for('esModuleInterop')` so that an import of `'module'` is not required.

Reply:
that was my first approach but it failed the tests. i speculate that symbols aren't shared between v8 contexts or something, if you have a solution please lemme know

Reply:
```js
> vm.runInNewContext('Symbol.for("foo")') === Symbol.for('foo')
true
```

I think this should work?